Java/Task1/src/PersonInformation.java

```diff
@@ -0,0 +1,2 @@
+public class PersonInformation {
```

Comment:
Please move you code in Main class here and delete Main class

Reply:
Could you see chenges please.